src/test/java/zipCodes/GetUserTest.java

```diff
@@ -0,0 +1,52 @@
+package zipCodes;
+
+import by.issoft.ResponseEntity;
+import by.issoft.client.UserClient;
+import by.issoft.dto.Sex;
+import by.issoft.dto.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class GetUserTest {
+
+    private static final int RESPONSE_CODE = 200;
+    private UserClient userClient;
+
+    @BeforeEach
+    void initUserClient() {
+        userClient = new UserClient();
+    }
+
+    @Test
+    void getUserTest() {
+        ResponseEntity<List<User>> users = userClient.getUsers();
+        Assertions.assertEquals(RESPONSE_CODE, users.getStatusCode());
+        Assertions.assertNotNull(users.getBody());
+    }
+
+    @Test
+    void getOlderUsersTest() {
+        Integer ageToCheck = 40;
+        ResponseEntity<List<User>> users = userClient.getUsers("olderThan", ageToCheck.toString());
+        Assertions.assertEquals(RESPONSE_CODE, users.getStatusCode());
+        users.getBody().forEach(user -> Assertions.assertTrue(user.getAge() > ageToCheck));
```

Comment:
there is one more way to verify
`Assertions.assertTrue(users.getBody().stream().allMatch(user -> user.getAge() > ageToCheck));`